tests/test_parser.py

```diff
@@ -352,8 +352,6 @@ def test_parse_tz_name_zzz(self):
             self.parser.parse("2013-01-01 12:30:45.9-10", "YYYY-MM-DDZZZ")
 
     def test_parse_subsecond(self):
-        # TODO: make both test_parse_subsecond functions in Parse and ParseISO
```

Comment:
Out of date I guess?

Reply:
Yeah I wanted to do a fixture for this in the past, but I don't know if it would work as well.